proto/go.mod

```diff
@@ -1,14 +1,24 @@
 module github.com/ory/keto/proto
 
-go 1.16
+go 1.19
 
 require (
-	github.com/stretchr/testify v1.8.0
-	golang.org/x/net v0.0.0-20220622184535-263ec571b305 // indirect
-	golang.org/x/sys v0.0.0-20220622161953-175b2fd9d664 // indirect
-	google.golang.org/genproto v0.0.0-20220622171453-ea41d75dfa0f // indirect
-	google.golang.org/grpc v1.49.0
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.15.0
```

Comment:
Why is this dependency added now? :thinking:

Reply:
fixed, I had stale generated files in `./proto`.